libs/features/mas/web-components/src/variants/variant-layout.js

```diff
@@ -0,0 +1,103 @@
+import { html } from 'lit';
+
+export class VariantLayout {
+  static styleMap = {};
```

Comment:
nit: I tend to use module scoped variants instead of static members.
Especially if they are meant to be `private`